### ISSUE TITLE
Add lists of cask, pypi, gems, and npm essentials

### ISSUE THREAD

**Issue by zmwangx** [zmwangx/dotfiles#1]

**Comment by zmwangx** [comment]
Beware that some casks need to be moved out of `/opt/homebrew-cask` to fully function (due to sandboxing issues), e.g., browsers (for 1Password extension):

```
chromium
firefox
firefox-beta
firefoxdeveloperedition
google-chrome
google-chrome-beta
google-chrome-dev
opera
opera-beta
opera-developer
```

**Comment by zmwangx** [comment]
`brew-cask` related stuff added in 1f38a3c8763a3ac30262cf56ca93df272a802787.

**Comment by zmwangx** [comment]
Fixed as of c637220c9eb3e8aae1ee88a6b370930e080afdae.
